finat/sympy2gem.py

```diff
@@ -65,3 +65,51 @@ def sympy2gem_symbol(node, self):
 @sympy2gem.register(symengine.Rational)
 def sympy2gem_rational(node, self):
     return gem.Division(*(map(self, node.as_numer_denom())))
+
+
+@sympy2gem.register(sympy.Abs)
+@sympy2gem.register(symengine.Abs)
+def sympy2gem_abs(node, self):
+    return gem.MaxValue(*map(self, node.args), gem.Literal(0))
+
+
+@sympy2gem.register(sympy.Equality)
+def sympy2gem_eq(node, self):
+    return gem.Comparison("==", *map(self, node.args))
+
+
+@sympy2gem.register(sympy.StrictGreaterThan)
+def sympy2gem_gt(node, self):
+    return gem.Comparison(">", *map(self, node.args))
+
+
+@sympy2gem.register(sympy.GreaterThan)
+def sympy2gem_ge(node, self):
+    return gem.Comparison(">=", *map(self, node.args))
+
+
+@sympy2gem.register(sympy.StrictLessThan)
+def sympy2gem_lt(node, self):
+    return gem.Comparison("<", *map(self, node.args))
+
+
+@sympy2gem.register(sympy.LessThan)
+def sympy2gem_le(node, self):
+    return gem.Comparison("<=", *map(self, node.args))
+
+
+@sympy2gem.register(sympy.Piecewise)
+@sympy2gem.register(symengine.Piecewise)
+def sympy2gem_conditional(node, self):
+    expr = None
+    pieces = []
+    for v, c in node.args:
+        if c == True:  # noqa: E712
+            expr = self(v)
```

Comment:
```suggestion
           expr = self(v)
           break
```

So that you don't generate more expression than you need to (this also matches the documented semantics of sympy.Piecewise